.github/workflows/publish_npm.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish schemas to npm
```

Comment:
Shouldn't this entire workflow be limited to running only on main?

We only want to publish at that point, right? Or, are we publishing dev versions outside main and it's okay to always publish?

Shouldn't we still limit publishing to main or open PRs, to avoid publishing constantly?

Reply:
If we ever wanted to have a beta release, it would be nice to be able to work on it in a separate branch and publish prereleases from there.